src/components/header/renderer.tsx

```diff
@@ -165,6 +203,35 @@ class Header extends React.PureComponent<Props, State> {
       </div>
     );
 
+    const authButton = (
+      <div className="auth-button-container">
+        {this.props.isAuthenticated ? (
+          <form action={`${BACKEND_URL}auth/github/logout`} method="get">
+            <div className="profile-container">
```

Comment:
That's a lot of classes. Do we need them all?

Reply:
@domoritz yeah, the component is a bit complicated

Reply:
@domoritz I removed a class called `profile-options`.